ITechArt.SurveysCreator.WebApp/ViewModels/SignUpViewModel.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace ITechArt.SurveysCreator.WebApp.ViewModels
+{
+    public class SignUpViewModel
+    {
+        [Required]
+        [Display(Name = "First Name")]
+        public string FirstName { get; set; }
+
+        [Required]
+        [Display(Name = "Second Name")]
+        public string SecondName { get; set; }
+
+        [Required]
+        [EmailAddress(ErrorMessage = "Current email is not valid!")]
+        [Remote("IsUnique", "Account", HttpMethod = "POST", ErrorMessage = "Some user with this email is already signed up!")]
+        public string Email { get; set; }
+
+        [Required]
+        [DataType(DataType.Password)]
+        public string Password { get; set; }
```

Comment:
Could you apply some policies for the password (e.g. contain uppercase,  lowercase)?